src/css/components/complex_list.scss

```diff
@@ -76,3 +76,6 @@ $color-border: $color-textblack;
   }
 }
 
+.complex_list-item.saving {
```

Comment:
@rememberlenny Same spacing question as below (line 88 of the other file in this PR).